.mods/update_mods.sh

```diff
@@ -28,6 +27,22 @@ declare -a MODS=(
 		"https://thunderstore.io/c/lethal-company/p/Hardy/LCMaxSoundsFix/"
 		"https://thunderstore.io/c/lethal-company/p/Clementinise/CustomSounds/"
 	"https://thunderstore.io/c/lethal-company/p/Owen3H/IntroTweaks/"
+	"https://thunderstore.io/c/lethal-company/p/Sligili/More_Emotes/"
+		"https://thunderstore.io/c/lethal-company/p/Ozone/Runtime_Netcode_Patcher/"
+	"https://thunderstore.io/c/lethal-company/p/LethalPlugging/LethalHaptics/"
```

Comment:
Where can I buy compatible devices? Asking for a friend of course :D

Reply:
https://buttplug.io/ take a look at the first paragraph of the underlying lib, they got a compatibility index as well.